assets/training/model_management/environments/foundation-model-inference/context/score.py

```diff
@@ -626,6 +636,11 @@ def build_chat_completion_prompt(data: List[str]) -> dict:
     return result
 
 
+def _sanitize_content(content: str) -> None:
```

Comment:
nit: the function is named like it should return a sanitized string.. how about add the strip part here, and let it return that (and raise if token in content like you have?)

Reply:
renamed function to reflect the logic better